### ISSUE TITLE
dialog component accessibility

### ISSUE THREAD

**Issue by goetsu** [AustinGil/vuetensils#104]
to improve accessibility of the dialog component https://vuetensils.stegosource.com/components/Dialog.html#installation
it would be great to add two attribute on on the generated <div tabindex="-1" role="dialog" class="vts-dialog__content"> <div class="bg-white">
*  aria-modal="true" that will make content outside of the modal not readable by screenreader
* aria-label="xxx" where xxx = a variable that we can configure for each modal to give it a name

**Comment by Stegosource** [comment]
Thanks. Ill add it to the list of fixes for the v1 release.
